indico_themes_canonical/static/css/ubuntu-summit/_forms.css

```diff
@@ -67,38 +67,45 @@ input[type]:not([type=number]):not([type=checkbox]):not([type=radio]):not([type=
 /* Style for radio buttons */
 .i-radio input {
   height: 1rem;
-  width: 1rem;
   margin-top: 5px;
+  width: 1rem;
```

Comment:
As a suggestion, maybe refactoring this with a variable could be easier? Line 50 also uses `width: 1rem`